src/reducer.init.js

```diff
@@ -0,0 +1,15 @@
+import { applyMiddleware, createStore } from 'redux';
```

Comment:
_Comment unrelated to line number_

I like the idea of a init reducer file, but maybe a better organization would be to create a `reducers/` folder with a `index.js` with this file contents

Later on we can start organizing the reducers in different files instead of putting them all in a single `reducer.js` file

Reply:
Refactored at e34cf6d152ef61784c3786e9f6805bdf8fb5052c